### ISSUE TITLE
[Accessibility]: Should blob view be formatted as a table?

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#36521]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34448

### Problem description

The blob view formats the code as a table, which feels incorrect; it doesn't match editors like GoLand. This means that with a screen reader, Control + Option + → alternates between going to the column numbers in the gutter and the code itself.

It is possible to vertically navigate the code using Control + Option + ↓ but that seems a little odd.

### Expected behavior

I'm not 100% sure, but it seems like having Control + Option + → go to the next line of code would be more intuitive.

In an editor like GoLand, you have an actual cursor which you can move up and down and if you're at the start of the line, VoiceOver will read the line's contents. So maybe it is OK to have Control + Option + ↓ be required?

### Additional details

If we decide to change this, it will require backend changes as we generate the HTML table in the backend. There are multiple code paths (see `generatePlainTable` and callers), and some of the HTML generation happens in the syntax highlighter.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by felixfbecker** [comment]
This is no longer the case in the CodeMirror blob view.
